extensions/stackdriver/log/exporter.cc

```diff
@@ -48,7 +48,8 @@ ExporterImpl::ExporterImpl(
   context_ = root_context;
   success_callback_ = [this](size_t) {
     logDebug("successfully sent Stackdriver logging request");
-    if (is_on_done_) {
+    in_flight_export_call_ -= 1;
+    if (in_flight_export_call_ <= 0 && is_on_done_) {
```

Comment:
< 0 should be an assertion no?
Not suggesting a real assert, but at least a warning ?

Reply:
Added a warn log about negative in flight request.